index.js

```diff
@@ -35,47 +45,73 @@
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
 
-// console.log(sameSameButDifferent(5, 5)) 
+function sameSameButDifferent(x,y) {
+  if (x === y) {
+    return "same same"
+  }
+  else if (x == y) {
+    return "same same (but different)"
+  }
+  else {
+    return "different"
+  }
+}
+
+console.log(sameSameButDifferent(5, 5)) 
```

Comment:
This is awesome! For the future tho I would change the parameter names to names that are a little more descriptive to what you're passing in (it's totally good for this but just a heads up for when you start coding your own projects!)